src/content/docs/integrations/kubernetes-integration/installation/install-fargate-integration.mdx

```diff
@@ -0,0 +1,392 @@
+
+```
+  $ kubectl rollout restart deployment <var>YOUR_DEPLOY</var>
+```
+
+#### Uninstall the Fargate integration
+
+We recommend that you uninstall the infrastructure operator that performs the automatic injection but keep the rest of the New Relic solution. Using Helm, disable the infra-operator by setting `infra-operator.enabled` to `false`, either in the `values.yaml` file or in the command line `--set`, and re-run the upgrade `--install` command above.
+
+To uninstall the whole solution:
+
+1. Completely uninstall the Helm release.
+2. Rollout the pods in order to remove the sidecar:
+    ```
+    $ kubectl rollout restart deployment <var>YOUR_APP</var>
+    ```
+3. Garbage collect the secrets:
+    ```
+    $ kubectl delete secrets -n <var>YOUR_NAMESPACE</var> -l newrelic/infra-operator-created=true
+    ```
+#### Known limitations: automatic injection
+
+Here are some issues to be aware of when using automatic injection:
+
+1. Currently there is no controller that watches the whole cluster to make sure that secrets that are no longer needed are garbage collected. However, all objects share the same label that can be used in order to remove all resources if needed `newrelic/infra-operator-created: true`.
+
+2. At the moment, it is not possible to use the injected sidecar to monitor services running in the pod. The sidecar will only monitor Kubernetes itself. However, if you are an advanced user, you may want to exclude these pods from automatic injection. You would then manually inject a customized version of the sidecar with on-host integrations enabled by configuring them and mounting their configurations in the proper place. [MISSING LINK].
+
+### Manual injection [#manual]
+If you have  any concerns about the automatic injection, you can inject the sidecar manually directly by modifying the manifests of the workloads scheduled that are going to be scheduled on Fargate nodes Please note that adding the sidecar into deployments scheduled into EC2 nodes may lead into incorrect or duplicate data, especially if those nodes are already being monitored with the DaemonSet.
+
+The following objects are required for the sidecar to successfully report data:
+
+* The `ClusterRole` providing the permission needed by the `nri-kubernetes integration`
+* A `ClusterRoleBinding` linking the `ClusterRole` and the service account of the pod
+* The secret storing the New Relic `licenseKey` in each Fargate namespace 
+* The sidecar container in the spec template of the monitored workload
+
+You can see example manifests for these objects in this [gist](https://gist.github.com/paologallinaharbur/754c13d0541f89d60ae74fdf877e3ce5J).
```

Comment:
This gist no longer exists.

Reply:
I believe we should not provide a gist, we should either put it into the repo or even better (IMO) in the docs themself

Reply:
By the way the link is "valid", there was an extra P in the url `https://gist.github.com/paologallinaharbur/754c13d0541f89d60ae74fdf877e3ce5 `

Reply:
Per @paologallinaharbur, I will remove the link and insert the snippets directly  in the doc.